packages/components/src/components/select/select.stories.ts

```diff
@@ -71,600 +61,314 @@ export default {
+        <sd-option value="option-1">Option 1</sd-option>
+        <sd-option value="option-2">Option 2</sd-option>
+        <sd-option value="option-3">Option 3</sd-option>
+        <sd-option value="option-4">Option 4</sd-option>
+        <sd-option value="option-5">Option 5</sd-option>
+      </sd-select>
+
+      <sd-select size="lg" label="Label" placement="top" value="">
+        <sd-option value="option-1">Option 1</sd-option>
+        <sd-option value="option-2">Option 2</sd-option>
+        <sd-option value="option-3">Option 3</sd-option>
+        <sd-option value="option-4">Option 4</sd-option>
+        <sd-option value="option-5">Option 5</sd-option>
+      </sd-select>
+    </div>
+  `
 };
 
 /**
- * `sd-select` with valid and invalid styles.
+ * Use the `label` attribute to provide an accessible label for the select element.<br />Use the `label` slot to add custom markup.
  */
 
-export const ValidInvalid = {
-  name: 'Valid x Invalid',
-  parameters: {
-    controls: {
-      exclude: ['label', 'open-attr', 'required', 'default', 'useTags', 'multiple', 'max-options-visible']
-    }
-  },
-  render: (args: { 'open-attr'?: string }) => {
-    delete args['open-attr'];
-
-    return html`<form class="h-[260px] w-full flex gap-4">
-      ${generateTemplate({
-        options: {
-          classes: 'w-full [&>tbody>tr>td]:align-top'
-        },
-        axis: {
-          y: {
-            type: 'attribute',
-            name: 'useTags',
-            values: [false, true]
-          },
-          x: {
-            type: 'attribute',
-            name: 'value',
-            values: ['option-1 option-2', '']
-          }
-        },
-        constants: [
-          twoOptionsConstant,
-          labelConstant,
-          multipleConstant,
-          { type: 'attribute', name: 'required', value: true }
-        ],
-        args
-      })}
-      <sd-button class="hidden" type="submit">Submit</sd-button>
-    </form>`;
-  },
-  play: async ({ canvasElement }: { canvasElement: HTMLUnknownElement }) => {
-    const el = canvasElement.querySelector('sd-button');
-    await waitUntil(() => el?.shadowRoot?.querySelector('button'));
-    await userEvent.type(el!.shadowRoot!.querySelector('button')!, '{return}', { pointerEventsCheck: 0 });
-  }
+export const Label = {
+  render: () => html`
+    <div class="w-[400px] h-[500px]">
+      <sd-select size="lg" label="Label" placement="bottom" value="">
+        <sd-option value="option-1">Option 1</sd-option>
+        <sd-option value="option-2">Option 2</sd-option>
+        <sd-option value="option-3">Option 3</sd-option>
+        <sd-option value="option-4">Option 4</sd-option>
+        <sd-option value="option-5">Option 5</sd-option>
+      </sd-select>
+    </div>
+  `
 };
 
 /**
- * Shows available slots. The `label` and `help-text` slots will overwrite their corresponding attributes.
+ * Use the `placeholder` attribute to show a placeholder when no option is selected.
```

Comment:
One the sample instead of "Place example" put "Placeholder example"

Reply:
Updated (this one was weird!)